.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: Release
+
+env:
+  FORCE_COLOR: 2
+  NODE: 14
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE }}
+
+      - name: Install dependencies and build
+        run: yarn install
+
+      - name: Run tests
+        run: yarn test
+
+      - name: Create Release Pull Request or Publish to npm
+        uses: changesets/action@master
+        with:
+          # this expects you to have a script called release which does a build for your packages and calls changeset publish
+          publish: yarn release
+          version: yarn version-packages
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
@diegomura could you add `NPM_TOKEN` to github secrets? it should be an [automation token](https://github.blog/changelog/2020-10-02-npm-automation-tokens/) that github announce couple days ago

Reply:
`GITHUB_TOKEN` is provided on default, so you don't need to add it.